.github/workflows/test.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-2022, windows-2019, ubuntu-18.04, ubuntu-20.04]
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.8, 3.9, "3.10", "3.11"]
```

Comment:
```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

Reply:
That causes the runners to explode

Reply:
That sounds like a valid reason to not accept my change.